content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -0,0 +1,241 @@
+---
+reviewers:
+- jsafrane
+title: Create static Pods
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+
+*Static Pods* are managed directly by kubelet daemon on a specific node,
+without the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
+observing it.
+Unlike Pods that are managed by the control plane (eg, a
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}};
+instead, kubelet itself watches it and restarts it when it crashes.
```

Comment:
Unlike Pods that are managed by the control plane, static Pods are managed by the kubelet. The kubelet watches static Pods, and restarts them when they crash.

General note: Choose a way of referring to the kubelet, and then be consistent throughout the whole topic. I prefer "the kubelet", but some people use "kubelet". I'd recommend against "Kubelet".